modules/pipeline/providers/reconciler/legacy/reconciler/reconcile_data_process.go

```diff
@@ -225,14 +225,14 @@ func (r *Reconciler) reconcileSnippetTask(task *spec.PipelineTask, p *spec.Pipel
 	}
 
 	if err := r.updateStatusBeforeReconcile(*sp); err != nil {
-		rlog.PErrorf(p.ID, "Failed to update pipeline status before reconcile, err: %v", err)
+		rlog.PErrorf(p.ID, "Failed to update pipeline status before Reconcile, err: %v", err)
```

Comment:
It seems that you have used global replacement, changing the lowercase reconcile to the uppercase Reconcile

Reply:
adjusted all unnecessary replacement.